
From: Hans Reiser <reiser@namesys.com>

Reiser4 implements read without help of generic_file_read, but it uses
page_cache_readahead.  So, this patch makes page_cache_readahead
EXPROT_SYMBOL.  Future filesystems may find this patch useful as well.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/mm/readahead.c |    1 +
 1 files changed, 1 insertion(+)

diff -puN mm/readahead.c~reiser4-export-page_cache_readahead mm/readahead.c
--- 25/mm/readahead.c~reiser4-export-page_cache_readahead	Tue Aug 24 15:38:05 2004
+++ 25-akpm/mm/readahead.c	Tue Aug 24 15:38:05 2004
@@ -515,6 +515,7 @@ do_io:
 out:
 	return;
 }
+EXPORT_SYMBOL(page_cache_readahead);
 
 
 /*
_
